tracing/src/main/kotlin/com/slack/sgp/tracing/model/TagBuilder.kt

```diff
@@ -37,5 +37,5 @@ public interface TagBuilder : MutableList<KeyValue> {
     KeyValue(key = this, v_type = ValueType.BINARY, v_binary = value).also(::add)
 }
 
-internal fun tagBuilderImpl(): TagBuilder =
+public fun tagBuilderImpl(): TagBuilder =
```

Comment:
Thanks, been meaning to make this public. Let's rename it as well along the way - `newTagBuilder()`?